server/src/domain/asset/asset.service.ts

```diff
@@ -187,26 +190,50 @@ export class AssetService {
         await this.access.requirePermission(authUser, Permission.ARCHIVE_READ, [dto.userId]);
       }
     }
+
+    if (dto.withPartners && dto.isArchived !== false) {
+      throw new BadRequestException('withPartners is only supported for non-archived assets');
+    }
```

Comment:
Can you extend this check to not allow other user specific flags such as `isFavorite`, `isTrashed`, etc to be sent when `withPartners` is set? Ideally, we'd want the user to be able to view only what's available on the partners main timeline but not their archived or trashed assets or even their favorites.